src/rsz/src/EstimateWireParasitics.cc

```diff
@@ -219,6 +219,8 @@ Resizer::incrementalParasiticsBegin()
     break;
   case ParasiticsSrc::global_routing:
     incr_groute_ = new IncrementalGRoute(global_router_, block_);
+    // Don't print verbose messages for incremental routing
+    global_router_->setVerbose(false);
```

Comment:
Great catch! I believe many messages generated in incremental runs can make the runtime worse.